documentation.lisp

```diff
@@ -929,4 +929,13 @@ this will error if the quasiquote contains any form of lexical references
 that are unresolvable.")
 
   (function call-compile
-    "Compiles the form with muffled warnings and calls the resulting function."))
+    "Compiles the form with muffled warnings and calls the resulting function.")
+
+  (variable *status-indicators*
+    "A plist which maps status values to a strings used in reports. Should
+have the status values of :passed, :failed, :skipped, :tentative and
+:unknown.")
+
+  (function status-character
+    "Return the appropriate status indicator for a specific result status. See
+*STATUS-INDICATORS*."))
```

Comment:
No, this won't work, sorry.

As the other docstrings show, a cross-reference needs to be formatted on its own line, without a dot at the end:

```
bla bla bla

See REFERENCE
```

Reply:
Gotcha